src/care_radiology/migrations/0004_alter_scanprotocol_body_part_and_more.py

```
@@ -0,0 +1,43 @@
+        ),
+        migrations.AlterField(
+            model_name='template',
+            name='body_part',
+            field=models.CharField(max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='template',
+            name='modality',
+            field=models.CharField(max_length=255),
+        ),
+    ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)